build/configs/artik053/STDK/defconfig

```diff
@@ -1246,6 +1243,7 @@ CONFIG_STDK_MQTT_USE_SSL=y
 CONFIG_STDK_IOT_CORE_USE_MBEDTLS=y
 # CONFIG_STDK_IOT_CORE_CRYPTO_SUPPORT_RSA is not set
 CONFIG_STDK_IOT_CORE_CRYPTO_SUPPORT_ED25519=y
+CONFIG_MBEDTLS_USER_FILE="/root/tizenrt/external/stdk/tizenrt-wrapper/mbedtls_config.h"
```

Comment:
What would happen if we are not using docker?
Is it possible to use relative path here?

Reply:
as we use this Kconfig string type, which always have "", so we can't use a global path with <>, and could only use such kind of absolute full path.
if we use normal make command, this path need to be set as the absolute path in build env. , maybe we can add such description in Kconfig help

Reply:
I think relative path is better and there is no problem.

Reply:
updated and verified with relative path, misundertood here before, thanks to you all